js/product-page.js

```diff
@@ -46,24 +49,24 @@
   };
 
   const changeUrl = function(data) {
-    let drinkName = data["drinks"][productId].name;
+    let drinkName = data["drinks"][drinkId].name;
```

Comment:
we talked about this one
you should find element with this ID not get just number of element
use "find" dunction
or "findIndex" or "indexOf"
google and change this ASAP